waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/NullAuthorizationProvider.java

```diff
@@ -0,0 +1,60 @@
+package com.hotels.bdp.waggledance.mapping.model;
+
+import java.util.List;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.metastore.api.Database;
+import org.apache.hadoop.hive.ql.metadata.AuthorizationException;
+import org.apache.hadoop.hive.ql.metadata.HiveException;
+import org.apache.hadoop.hive.ql.metadata.Partition;
+import org.apache.hadoop.hive.ql.metadata.Table;
+import org.apache.hadoop.hive.ql.security.HiveAuthenticationProvider;
+import org.apache.hadoop.hive.ql.security.authorization.HiveAuthorizationProvider;
+import org.apache.hadoop.hive.ql.security.authorization.Privilege;
+
+public class NullAuthorizationProvider implements HiveAuthorizationProvider {
```

Comment:
What's this class needed for?

Reply:
I'll delete this. When I was first testing things I thought we needed to override the default authorizer, but that changed when we decided to load the filter ourselves.